test/libsolidity/InlineAssembly.cpp

```diff
@@ -671,12 +650,6 @@ BOOST_AUTO_TEST_CASE(for_statement)
 	BOOST_CHECK(successAssemble("{ let x := calldatasize() for { let i := 0} lt(i, x) { i := add(i, 1) } { mstore(i, 2) } }"));
 }
 
-BOOST_AUTO_TEST_CASE(if_statement)
-{
-	BOOST_CHECK(successAssemble("{ if 1 {} }"));
-	BOOST_CHECK(successAssemble("{ let x := 0 if eq(calldatasize(), 0) { x := 1 } mstore(0, x) }"));
```

Comment:
Does this test still exist?

Reply:
Yes, I just added it however to `if_statement.yul`. Thanks.